### DISCUSSION TITLE
Build with webpack

### DISCUSSION THREAD

**Discussion by gaeVG** [prisma/prisma#13611] [Q&A]
Hello,
I tried to integrate Prisma in my project to manage my database, and unfortunately at the compilation I have this error that I can't solve:

```
Module not found: Error: Can't resolve 'encoding' in 'node_modules\@prisma\client\runtime'
```

I have scrupulously followed the instructions in the documentation and yet nothing happens.

If someone would have the answer, it would help me a lot !

Thanks in advance 🙏

**Comment by austincrim** [comment]
Hey @gaeVG 👋,

Could you give more details about your setup? A minimal reproduction repo would be ideal, but if you can't provide that:

1. What database are you using?
2. What is your backend setup?
3. How are you using Webpack?

**Reply by gaeVG** [reply]
Hello, and sorry indeed the information is rather poor.
I have just created a [repository](https://github.com/gaeVG/ts_paradise) so that you can have all the necessary information.

Don't hesitate to ask me if you need more information.

**Reply by austincrim** [reply]
Is there any way you could narrow that repo down? There is a ton in there and it would speed things up a lot if you could trim away the unrelated bits.

**Reply by gaeVG** [reply]
Oh sorry ^^' I'll do it right away

**Reply by gaeVG** [reply]
Apart from a few configuration files, normally only what concerns us is left

**Reply by austincrim** [reply]
Thanks for the quick response! Although, I think I will at least need to have the files where you were actually using Prisma in your code and the entry files that Webpack was using.

**Reply by gaeVG** [reply]
I don't have any files where I actually use it yet, I tried to configure it and verify that it would compile correctly. And obviously it doesn't work well 😢

**Reply by pantharshit00** [reply]
Hello @gaeVG! Sorry for late follow up here.

Are you still able to reproduce this problem with the latest version? If so, can you please share a reproduction. Looks like you have deleted the repository that you have posted earlier.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
